test/spec/test-browser-specific.js

```diff
@@ -787,4 +787,24 @@ describe('tus', () => {
       await assertUrlStorage(tus.defaultOptions.urlStorage)
     })
   })
+
+  describe('#Checksum', () => {
```

Comment:
We would also need a full test to ensure the checksum header is added to the actual requests. Such a test should be placed in `test-common`, which includes tests shared between browsers and Node.js.

Let me know if you need help with getting this running :)